installer/charts/values.yaml.tpl

```diff
@@ -6,9 +6,12 @@
 {{- $gitops := required "GitOps settings" .Installer.Features.openShiftGitOps -}}
 {{- $pipelines := required "Pipelines settings" .Installer.Features.openShiftPipelines -}}
 {{- $quay := required "Quay settings" .Installer.Features.redHatQuay -}}
+{{- $nexusSecret := (lookup "v1" "Secret" .Installer.Namespace "rhtap-nexus-integration") -}}
+{{- $quaySecret := (lookup "v1" "Secret" .Installer.Namespace "rhtap-quay-integration") -}}
+{{- $quayEnabled := not (or $nexusSecret $quaySecret) -}}
```

Comment:
1. The variable name is confusing
2. If quay is deployed by the installer, the secret doesn't exist yet.

Reply:
Please revert